cypress/pages/template-collection.ts

```diff
@@ -0,0 +1,19 @@
+import { BasePage } from './base';
+
+export class TemplateCollectionPage extends BasePage {
```

Comment:
we are trying to move away from this pattern.. checkout `pages/demo.ts` for an example